src/game/game.py

```diff
@@ -1,13 +1,21 @@
 import numpy as np
 from threading import Thread, Timer
 from game.random import random_move
-from game.exception import MoveException, NoEmptyTileException, NoTileFoundException
+from game.exception import SizeNotValidException, MoveException, NoEmptyTileException, NoTileFoundException
 
 
-def build_grid(size=4):
+MIN_SIZE = 2
+MAX_SIZE = 10
+DEFAULT_SIZE = 4
+
+
+def build_grid(size=DEFAULT_SIZE):
+    if not MIN_SIZE <= size <= MAX_SIZE:
```

Comment:
Can you test the upper and lower bound error ?
I don't see this error on the test